### DISCUSSION TITLE
Not found issue

### DISCUSSION THREAD

**Discussion by vulnz** [Security-Onion-Solutions/securityonion#6307] [Unsupported Versions] [answered]
Good day guys.
Thanks for creating such a good product.
Please let me know on how to fix the following issue:
I try to access kibana and some other links and they happen to be 404.
Can you please provide me information on how to make them start?
I run command to start all services, but this still did not work.
Thanks

**Comment by dougburks** [comment]
From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:

Please include the following details where you can:

- Security Onion version as seen in the lower left corner of SOC and in /etc/soversion. For example: 2.3.80
- Is this a cloud deployment or on-prem?
- Did you install from our Security Onion ISO image or did you perform a network installation?
- If network installation, did you install on CentOS 7 or Ubuntu?
- How many nodes do you have? How are each of those nodes configured? (ex. manager with 2 search nodes and 3 forward nodes)
- Are you experiencing issues monitoring network traffic? If so, are you sniffing from a tap or span port and what is the traffic volume?
- Does so-status show all services running?
- Do you get any failures when you run sudo salt-call state.highstate?
- Does the SOC Grid page show any failures?
- Explain your issue. For example: Installation fails when I select this series of options...
- Provide applicable logs. For example: if you are having problems during soup, provide /root/soup.log

**Comment by vulnz** [comment]
Good day.

1) Version: 2.3.80
2) On prem
3) ISO
4) Downloaded from site iso
5) one node
6) I did not get to the level of traffic monitoring yet
7) Yes, so-status shows that all works
8) salt-call state.highstate - no issues running
9) SOC grid is main page? it works.
10) Issue is that Kibana and some other things show 404
11)  sudo salt-call state.highstate = no output, just hang terminal
12) Thehive is running but shows 404
13) The kibana sais - missing
14) if I run sudo so-start kibana --force it will stop some saltwork and after goes image error

Please let me know how to reconfigure the system so it will have all missing dependency.
thanks


![image](https://user-images.githubusercontent.com/20043135/143185721-b9f63c12-98d9-4529-ab2c-dd723f7e5cc2.png)

**Comment by dougburks** [comment, marked as answer]
Please try a new installation using the new 2.3.90 ISO:
https://blog.securityonion.net/2021/11/security-onion-2390-wazuh-hotfix-now.html

Also make sure you are following the hardware requirements:
https://docs.securityonion.net/en/2.3/hardware.html#standalone-deployments

**Comment by vulnz** [comment]
thanks, brother
